cmd/prometheus/main.go

```diff
@@ -217,7 +217,7 @@ func main() {
 	a.Flag("storage.tsdb.wal-compression", "Compress the tsdb WAL.").
 		Default("true").BoolVar(&cfg.tsdb.WALCompression)
 
-	a.Flag("storage.remote.flush-deadline", "How long to wait flushing sample on shutdown or config reload.").
+	a.Flag("storage.remote.flush-deadline", "How long to wait flushing sample and metadata on shutdown or config reload.").
```

Comment:
can we make this more generic just so we don't need to change this again when we add more data types to remote write in the future?

Reply:
Is that better?

Reply:
yep